charts/brigade-metrics/values.yaml

```diff
@@ -53,7 +53,7 @@ prometheus:
     tag: v2.28.0
     pullPolicy: IfNotPresent
 
-  scrapeInterval: 3
+  scrapeInterval: 3s
```

Comment:
See [this comment](https://github.com/willie-yao/brigade-metrics/pull/17/files#r663135043)